app/Http/Controllers/TagsController.php

```diff
@@ -37,13 +44,13 @@ public function __construct(PostRepository $post)
      */
     public function store(TagsRequest $request)
     {
-        $tagData = $request->all();
-        $taggedPost = $this->post->tag($tagData);
+        $post = $this->post->find($request->input('post_id'));
+        $taggedPost = $this->post->tag($post, $request->input('tag_name'));
+
+        // @TODO: $post isn't showing the updated tags right now.
 
-        if (! $taggedPost) {
-            response()->json(['error' => 'Tag was not successfully created/deleted.'], 500);
-        }
+        // @TODO: We should use a transformer anywhere we send data to client.
```

Comment:
Just wondering if this is for the future or if you meant to use the transformer here since I noticed above that you have one now